WTWSMS/history/characters/old_frankish.txt

```diff
@@ -1679,7 +1679,7 @@
 190426 = {
 	name="Suanachildis" #wife of charles the hammer
 	female=yes
-	dynasty=1044033 #Agilolfing
+	dynasty=1044125 #Agilolfing
```

Comment:
We said the ID was to be kept to avoid conflicts with vanilla.

Reply:
Oops, My bad didn´t change it here

Reply:
Solved